src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplDownloadSteps.java

```diff
@@ -51,13 +53,17 @@ public ChplDownloadSteps() {
         if (StringUtils.isEmpty(downloadPath)) {
             String tempDirectory;
             try {
-              tempDirectory = Files.createTempDirectory("download-files").toString();
-              System.out.println(tempDirectory);
+                tempDirectory = Files.createTempDirectory("download-files").toString();
+                // Print the path to the newly created directory
+                System.out.println(tempDirectory);
             } catch (final IOException e) {
-              // Temp dir creation failed - use default
-              tempDirectory = null;
+                // If temp directory creation failed, create new directory in target folder
+                File file = new File("target", "download-files");
+                file.mkdirs();
+                tempDirectory = System.getProperty("user.dir") + "\\target\\download-files";
+                System.out.println(tempDirectory);
```

Comment:
Where does `file.mkdirs();` make the files/directories? Is it in `user.dir`? Are you sure? Is there always a property called user.dir? Also, the `+ "\\target\\download-files";` only works on Windows. On Linux the directory separator is a forward slash `/`.  You should use `File.pathSeparator` instead.